frontend/staff-dashboard/src/pages/flexible_pricing/list.tsx

```diff
@@ -154,6 +159,11 @@ export const FlexiblePricingList: React.FC = () => {
                                 title="Date Calculated"
                                 render={(value) => <DateField format="LLL" value={value} />}
                             />
+                            <Table.Column
+                                dataIndex="discount"
+                                title="Discount"
+                                render={(value) => <div>{value.amount} {value.discount_type} {value.discount_code}</div>}
```

Comment:
See comment in the status modal about formatting - same thing should apply here.